src/test/content-structure.test.ts

```
@@ -0,0 +1,115 @@
+      expect(frTranslationId).toBeDefined();
+      expect(enTranslationId).toBe(frTranslationId);
+    });
+
+    it('should have valid dates', () => {
+      const firstPostPath = join(CONTENT_DIR, 'blog', 'en', 'first-post.mdx');
+      const content = readFileSync(firstPostPath, 'utf-8');
+      
+      const dateMatch = content.match(/pubDate: (\d{4}-\d{2}-\d{2})/);
+      expect(dateMatch).toBeDefined();
+      
+      const date = new Date(dateMatch![1]);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)